src/core/raster/qgsrasterlayer.h

```diff
@@ -76,6 +76,9 @@ typedef QList < QPair< QString, QColor > > QgsLegendColorList;
 class CORE_EXPORT QgsRasterLayer : public QgsMapLayer, public QgsAbstractProfileSource
 {
     Q_OBJECT
+
+    Q_PROPERTY( QString mapTipTemplate READ mapTipTemplate WRITE setMapTipTemplate NOTIFY mapTipTemplateChanged )
```

Comment:
is this actually needed? Wouldn't it be inherited from the QgsMapLayer class?

Reply:
Yep, leftover from a previous iteraation before I moved the API to the parent class.